src/js/components/WorldMap/index.d.ts

```diff
@@ -21,20 +21,20 @@ export interface WorldMapProps {
       | 'Europe'
       | 'North America'
       | 'South America';
-    onClick?: (...args: any[]) => any;
-    onHover?: (...args: any[]) => any;
+    onClick?: (name: string) => void;
+    onHover?: (hovered: boolean) => void;
   }[];
   fill?: FillType;
   gridArea?: GridAreaType;
   hoverColor?: string | { dark?: string; light?: string };
   margin?: MarginType;
-  onSelectPlace?: (place: number[]) => void;
+  onSelectPlace?: (place: [number, number]) => void;
   places?: {
     color?: string | { dark?: string; light?: string };
     name?: string;
     location: number[];
-    onClick?: (...args: any[]) => any;
-    onHover?: (...args: any[]) => any;
+    onClick?: (name: string) => void;
+    onHover?: (hovered: boolean) => void;
```

Comment:
I think those two should accept events.

Reply:
The reason I defined these like that is because theses handlers are being called with just the name of the place/continent (for onClick), and a boolean indicating weather or not hover is active (for onHover)

You can see it here: 
line 457, 462, 470 https://github.com/grommet/grommet/blob/c4300ef91515c44a51f3759dafbc1a7bf1b47849/src/js/components/WorldMap/WorldMap.js#L449-L473

Reply:
I'll create some tests with jest just to make sure they will fail if in the future these handlers are changed and we can update the type definition.

Regarding your comment, were you actually thinking we should change the implementation or just making sure the type is correct?

Reply:
Thanks! I just want us to make sure the type is correct.

Reply:
Added a couple tests

Reply:
I'm not sure how the tests help to answer the question, what am I missing?

Reply:
With the test I'm showing `onClick` only sends the name of the place, and nothing more. i.e. there is currently no event that can be added to the type definition.
```
expect(onPlaceClick).toHaveBeenCalledWith('Sydney');
```
Also, if the implementation ever changes, the test will fail which hopefully will be a reminder that the type definition must be updated at that point. 

The same logic applies for `onHover`.

Is this the type of answer you were looking for? Maybe I missed something and haven't answered the correct question.